packages/core/src/common/text/text.c

```diff
@@ -711,7 +717,9 @@ void comboTextAppendItemNameEx(char** b, s16 gi, int flags, int importance)
             itemName = "a " TEXT_C2 "Progressive Goron Lullaby";
             break;
         case GI_OOT_SCALE_SILVER:
+        case GI_MM_SCALE_SILVER:
             itemName = "a " TEXT_C1 "Progressive Scale";
+            ambiguous = !comboConfig(CFG_SHARED_SCALES);
```

Comment:
Needs to check for the active setting too (it's not ambiguous if MM scale isn't on)